client/src/components/CountdownTimer.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { useState, useEffect, useRef } from 'react';
+
+const CountdownTimer: React.FC = () => {
```

Comment:
add zero case handling: if the countdown reaches `0` as a use i like to show me something like: "Time's up!"
or disabling the `Start` button